README.md

```diff
@@ -33,6 +33,11 @@
 
 **Leaderboard** The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 
+<div align="center">
+
+  <img src="./app-screenshot.png" alt="app-screenshot" width="auto"  height="auto" />
+  </div>
+
```

Comment:
- [ ] The live version is different, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/209173320-18954807-0e76-4067-a369-c466ce65b6cf.png)

Reply:
Thanks @GzimAsani,
it's done now.